### ISSUE TITLE
deprecated multiplication operator

### ISSUE THREAD

**Issue by verybigelephants** [Klowner/inkscape-applytransforms#56]
When trying to apply transform i get a deprecated warning:

```
DeprecationWarning: inkex.deprecated.main.transform_mul -> Use @ operator instead
  transf = Transform(transf) * Transform(node.get("transform", None))
```

**Comment by jtojnar** [comment]
This is a duplicate of https://github.com/Klowner/inkscape-applytransforms/issues/55.

**Comment by vmario89** [comment]
i am going to fix that stuff in the next month for several other plugins; since python 3.10 and Inkscape 1.2 there are a lot of new deprecations in multiple extensions

**Comment by kongo09** [comment]
Any progress on this?

I did try to replace `*` with `@` in the code. The error disappears, but the result does not show a transform.

**Comment by Klowner** [comment]
If you're not seeing the results you expect, please open a new ticket with an example svg and steps to reproduce.

This deprecation **warning** is not an **error** and does not impact the operation of this extension. The warning lets us know that it will eventually break with a *future release of inkscape*. When that time comes, there will not be a **warning** but an actual **error**.

**Comment by aaronfay** [comment]
@Klowner understood, however the extension is still not flattening the SVG. I will open a new ticket with more details.

**Comment by JapanYoshi** [comment]
Just got this same error on Inkscape 1.3

```
C:\Users\Haley\AppData\Roaming\inkscape\extensions\applytransform.py:60: DeprecationWarning: inkex.deprecated.main.transform_mul -> Use @ operator instead
  transf = Transform(transf) * Transform(node.get("transform", None))

```

**Comment by ziegenberg** [comment]
Fixed by #62, can be closed now.
